### PR TITLE
[ZIPFLDR] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

**Review by JoachimHenze — APPROVED** [review]

File: dll/shellext/zipfldr/lang/ro-RO.rc [hunk 1]
```diff
@@ -13,10 +13,10 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Selectați o destinație"
 FONT 8, "MS Shell Dlg", 400, 0, 0x0
 BEGIN
-    LTEXT           "Selectați folderul de destinație",IDC_STATIC,86,12,154,8
+    LTEXT           "Selectaţi un folder unde vor fi extrase fişierele.",IDC_STATIC,86,12,154,8
```
Comment (JoachimHenze) [inline dll/shellext/zipfldr/lang/ro-RO.rc:16]: does that fit the button? or is it cut off by the 154?

Comment (AndreiMiloiu) [reply dll/shellext/zipfldr/lang/ro-RO.rc:16]: You're right. I made a mistake. I will revert this change.

Comment (JoachimHenze) [reply dll/shellext/zipfldr/lang/ro-RO.rc]: Resolved, thanks.

**Comment by icf20** [comment]
folder nu e dosar ?

**Comment by AndreiMiloiu** [comment]
> folder nu e dosar ?

This is the translation recommended by MSDN. They translate folder and not dosar.

**Pull request merged**